app/helpers/products_helper.rb

```diff
@@ -25,10 +25,12 @@ def options_for_relay
     }
   end
 
-  def public_calendar_link(product)
+  def public_calendar_link(product, should_show_icon)
     if product.respond_to? :reservations
       opts = public_calendar_options(product)
       link_to "", facility_instrument_public_schedule_path(product.facility, product), opts
+    elsif should_show_icon
+      content_tag :span, "", class: "fa-lg fa-fw", style: "display: inline-block"
```

Comment:
We don't have to, but it might be nice to get the `style` piece out of this and into CSS. I originally suggested it here just as a quick and dirty way to test it.

Reply:
Honestly, this whole thing is kind of a hack, but I think it's more clear to keep it all together in one place rather than extracting this super odd special-case CSS into a separate file somewhere.